### PR TITLE
NERD and SEER ODS Automation ~Maria

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

File: src/main/resources/.DS_Store
Comment (matakevin) [inline src/main/resources/.DS_Store:1]: Can you remove this file from the PR?  We don't need to push this

File: src/test/java/ServiceNow/COVIDDash/Resources/.DS_Store
Comment (matakevin) [inline src/test/java/ServiceNow/COVIDDash/Resources/.DS_Store:1]: Remove this file also

File: src/test/java/ServiceNow/PlatformBusinessApps/NERD/Constants/Covid19ActivitiesSubmissions_Constants.java
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/NERD/Constants/Covid19ActivitiesSubmissions_Constants.java:1]: remove empty lines 39 and 40 please

File: src/test/java/ServiceNow/PlatformBusinessApps/NERD/Pages/CreateNewSubmissionPage.java
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/NERD/Pages/CreateNewSubmissionPage.java:1]: remove line 240

File: src/test/java/ServiceNow/PlatformBusinessApps/NERD/Pages/NERDDynamicXPATHS.java
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/NERD/Pages/NERDDynamicXPATHS.java:1]: remove lines 221 and 226

**Review by georgeb79 — APPROVED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/NERD/Steps/Covid19ActivitiesSubmissionsSteps.java
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/NERD/Steps/Covid19ActivitiesSubmissionsSteps.java:1]: remove line 27.  Lines 17-21 look formatted oddly, if that's how it's displaying for you then can you format and push again?

File: src/test/java/ServiceNow/PlatformBusinessApps/NERD/StepsImplementation/Covid19ActivitiesSubmissionsStepImplementation.java
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/NERD/StepsImplementation/Covid19ActivitiesSubmissionsStepImplementation.java:1]: All you methods in the stepImpl classes should have a javadoc or comment stating what they're doing

**Review by mzuhoor — APPROVED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/SEER/Features/DUA_Agreements.feature
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/SEER/Features/DUA_Agreements.feature:1]: remove progression tab from Pull Requests

File: src/test/java/ServiceNow/ServiceNowRunners/NERDRunners/Run_NERD_Progression_Test.java
Comment (matakevin) [inline src/test/java/ServiceNow/ServiceNowRunners/NERDRunners/Run_NERD_Progression_Test.java:1]: I could be wrong but the .html is needed to have comments in reports, please check with Diego

File: src/test/java/ServiceNow/ServiceNowRunners/SEERRunners/Run_SEER_Regression_Test.java
Comment (matakevin) [inline src/test/java/ServiceNow/ServiceNowRunners/SEERRunners/Run_SEER_Regression_Test.java:1]: We are no longer using JUnit.  Please change this back to the TestNG CucumberOptions import

**Review by matakevin — CHANGES_REQUESTED** [review]
Let some changes that need to be made or looked at.

**Review by mzuhoor — APPROVED** [review]

**Review by mzuhoor — APPROVED** [review]

**Review by matakevin — APPROVED** [review]
LGTM

**Review by mzuhoor — APPROVED** [review]

**Review by mzuhoor — APPROVED** [review]

**Review by alenaminsk — APPROVED** [review]

**Pull request merged**